openwisp_users/tests/utils.py

```diff
@@ -76,6 +89,8 @@ def _test_multitenant_admin(self, url, visible, hidden, select_widget=False):
         a superuser can see everything.
         """
         self._login(username='operator', password='tester')
+        if administrator:
+            self._login(username='administrator', password='tester')
```

Comment:
this is weird here, we either log in as operator or administrator, not both, it may not do anything if we log in first with operator and then with administrator, but it's not clean, so please clean it up

Reply:
Resolved.